_posts/2022-12-05-solidity-core-team-updates.md

```diff
@@ -0,0 +1,142 @@
+
+#### What is your role in the Solidity team? On which parts of Solidity do you work the most?
+
+*I work primarily on Solc-js (a JavaScript wrapper around the Solidity compiler), task automation, and Continuous Integration scripts in the Solidity repository. I also collaborate with the development of some minor changes in the compiler, and I'm looking forward to contributing to more compiler features and optimizations in future.*
+
+#### What made you join the Solidity team?
+
+*Being part of the Solidity team sounded like an exciting challenge to me - a natural step for learning more about the Solidity language, and an excellent way to contribute and learn more about the Ethereum ecosystem.*
+
+## Changes in Team Structure 🧑‍💻👩‍💻
+
+We also want to use this post as an opportunity to share insights into recent updates to our team structure. We believe that the new team structure and processes will support us to productively deliver on our goals and focus on our priorities. Moving forward, the team leadership consists of a Technical Product Manager and a Tech Lead.
+
+### Introducing Nuno as Solidity Product Manager 🎯
+
+End of September, we were happy to onboard Nuno as our new Technical Product Manager. Nuno supports the team with leading people and processes, as well as facilitating the product/roadmap planning and overall development coordination. Nuno has worked with open-source projects all of his life and we are inspired to learn from and with him!
+
+### Welcoming Daniel (@ekpyron) as new Solidity Tech Lead 🧭
+
+Furthermore, we'd like to announce that one of our long-time Solidity core team members, Daniel, has stepped up to become Solidity's next Tech Lead.
+
+Daniel started contributing to Solidity in February of 2018. He is a mathematician by training, but already worked as a part-time software developer ever since his studies. This year, he received a PhD in Computational Metaphysics.
+
+Daniel demonstrated technical leadership throughout his work on the optimizer and viaIR pipeline in Solidity and has been a vital sparring partner for language design decisions since he joined the team.
+
+We are glad to have him as a new Tech Lead and confident he is the perfect choice to continue Chris' legacy.
+
+If you want to hear more from Daniel, consider checking out his talk on ["Generating EVM Bytecode from Yul in the new via IR Pipeline"](https://www.youtube.com/watch?v=RJQdycaEgIE) at this year's Solidity Summit or a more general update on ["Technical Details of the Solidity Compiler"](https://archive.devcon.org/archive/watch/6/technical-details-of-the-solidity-compiler/?tab=YouTube) at Devcon.
```

Comment:
```suggestion
Furthermore, we'd like to announce that one of our long-time Solidity core team members, Daniel, has stepped up to become Solidity's next Tech Lead.

Daniel started contributing to Solidity in February 2018. He is a mathematician by training, but he has worked as a part-time software developer since his studies. This year, he received a Ph.D. in Computational Metaphysics.

Daniel demonstrated technical leadership throughout his work on the optimizer and via-IR pipeline in Solidity and has been a vital sparring partner for language design decisions since he joined the team.

We are glad to have him as a new Tech Lead and confident he is the perfect choice to continue Chris' legacy.

If you want to hear more from Daniel, consider checking out his talk on ["Generating EVM Bytecode from Yul in the new via IR Pipeline"](https://www.youtube.com/watch?v=RJQdycaEgIE) at this year's Solidity Summit or a more general update on ["Technical Details of the Solidity Compiler"](https://archive.devcon.org/archive/watch/6/technical-details-of-the-solidity-compiler/?tab=YouTube) at Devcon.
```